src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/RazorFormattingOptions.cs

```diff
@@ -15,6 +16,8 @@ internal readonly record struct RazorFormattingOptions
     public int TabSize { get; init; } = 4;
     [DataMember(Order = 2)]
     public bool CodeBlockBraceOnNextLine { get; init; } = false;
+    [DataMember(Order = 3)]
+    public RazorCSharpSyntaxFormattingOptions? CSharpSyntaxFormattingOptions { get; init; }
```

Comment:
>{ get; init; } [](http://example.com/codeflow?start=77&length=14)

We are fine with all defaults here? No need to assign explicitly?

Reply:
Yeah, `null` is fine for the default and matches existing behaviour. In future we can probably clean it up more, if we remove the old LSP, and we can roll out proper formatting options support to code actions and auto insert etc. but no need to open that can of worms now.